src/promptflow-core/promptflow/core/_prompty_utils.py

```diff
@@ -15,6 +16,41 @@
 from promptflow.core._utils import render_jinja_template_content
 
 
+@dataclass
+class PromptyModelConfiguration:
+    api: str
+    configuration: dict
```

Comment:
are we plan to add the AOAIModelConfig class in next PR?

Reply:
add AzureOpenAIModelConfiguration and OpenAIModelConfiguration